examples/nimble-typescript/README.md

```diff
@@ -0,0 +1,117 @@
+# Nimble Typescript Getting Started
+
```

Comment:
Could you add a link to this example from the `nimble-components` README?

We could also consider linking to it from nimble.ni.dev but that doesn't need to happen now if the vision for that page is still being established.

Reply:
Done
[4296847](https://github.com/ni/nimble/pull/796/commits/4296847f0fe32ecef779f97ac7931ea10c90aba1)